src/util/nb_native.js

```diff
@@ -82,7 +83,8 @@ async function read_rand_seed(seed_bytes) {
     while (offset < buf.length) {
         try {
             const count = buf.length - offset;
-            const random_dev = process.env.DISABLE_DEV_RANDOM_SEED ? '/dev/urandom' : '/dev/random';
+            if (process.env.DISABLE_DEV_RANDOM_SEED === undefined) process.env.DISABLE_DEV_RANDOM_SEED = config.DISABLE_DEV_RANDOM_SEED;
+            const random_dev = process.env.DISABLE_DEV_RANDOM_SEED === 'true' ? '/dev/urandom' : '/dev/random';
```

Comment:
Will this work on containerized NooBaa as well?
I'm asking because the previous check was -
`const random_dev = process.env.DISABLE_DEV_RANDOM_SEED ? '/dev/urandom' : '/dev/random';`
and now it was changed to -
`rocess.env.DISABLE_DEV_RANDOM_SEED = config.DISABLE_DEV_RANDOM_SEED;
            const random_dev = process.env.DISABLE_DEV_RANDOM_SEED === 'true' ? '/dev/urandom' : '/dev/random';`
so we no longer check for boolean value but for string 'true'

Reply:
@romayalon ` process.env.DISABLE_DEV_RANDOM_SEED` is a type of `string`,  This condition `process.env.DISABLE_DEV_RANDOM_SEED ?` always  return `true` when  `DISABLE_DEV_RANDOM_SEED` is not empty, So even if `process.env.DISABLE_DEV_RANDOM_SEED` is `false`, This condition will assign `/dev/urandom` to random_dev.

Reply:
I was thinking about a situation where - 
process.env.DISABLE_DEV_RANDOM_SEED = 'false';
Before the change- 
random_dev = '/dev/urandom' ;
After - 
random_dev = '/dev/random';

Reply:
@naveenpaul1 but I'm starting to think that before the change it was a bug...